content/influxdb/v2.0/query-data/influxql.md

```diff
@@ -16,46 +16,101 @@ related:
+- To find a specific bucket (`bucket_id`), database (`database`), retention policy (`retention_policy`), or mapping ID (`id`), include the parameter in your request.
+
+```sh
+curl -GET https://cloud2.influxdata.com/api/v2/dbrps \
+  -H "Authorization: Token YourAuthToken" \
+  -H 'Content-type: application/json' \
+  -d '{
+       "bucket_id": "12ab34cd56ef",
+       "database": "example-db",
+       "id": "example-mapping-id"
+       "default": true
+       "organization_id": "example-org",
+     }'
+```
+
+If you **do not find a mapping ID (`id`) for a bucket**, complete the next procedure to map the unmapped bucket.
+
+## Map unmapped buckets
+
+To map an unmapped bucket to a database and retention policy, use the [`POST /dbrps` API request](/influxdb/v2.0/api/#operation/PostDBRP) (see CURL example below).
+
+ You must include an **authorization token** with [basic or token authentication](/influxdb/v2.0/reference/api/influxdb-1x/#authentication) in your request header and the following **required parameters** in your request body:
+
+ - organization (`organization` or `organization_id`)
+ - target bucket (`bucket_id`)
+ - database and retention policy to map to bucket (`database` and `retention_policy`)
+
+```sh
+curl -XPOST https://cloud2.influxdata.com/api/v2/dbrps \
+  -H "Authorization: Token YourAuthToken" \
+  -H 'Content-type: application/json' \
+  -d '{
+       "bucket_id": "12ab34cd56ef",
+       "database": "example-db",
+       "default": true
+       "organization": "example-org",
+       "organization_id": "example-org",
+       "retention_policy": "example-rp",
+      }'
+```
+
+After you've verified the bucket is mapped, query the bucket using the `query` 1.x compatibility endpoint.
+
+## Query a mapped bucket with InfluxQL
+
 The [InfluxDB 1.x compatibility API](/influxdb/v2.0/reference/api/influxdb-1x/) supports
 all InfluxDB 1.x client libraries and integrations in InfluxDB 2.0.
 
-Provide the following:
+To query a mapped bucket with InfluxQL, use the `/query` 1.x compatibility endpoint (see CURL example below), and include the following in your request:
 
 - InfluxDB [authentication token](/influxdb/v2.0/security/tokens/)
   _(See [compatibility API authentication](/influxdb/v2.0/reference/api/influxdb-1x/#authentication))_
 - **db query parameter**: 1.x database to query
-- **rp query parameter**: 1.x retention policy to query
+- **rp query parameter**: 1.x retention policy to query; if no retention policy is specified, InfluxDB uses the default retention policy for the specified database.
 - **q query parameter**: InfluxQL query
 
 {{% note %}}
 **URL-encode** the InfluxQL query to ensure it's formatted correctly when submitted to InfluxDB.
 {{% /note %}}
 
 ```sh
-curl -G https://cloud2.influxdata.com/query?db=db&rp=rp \
+curl -G https://cloud2.influxdata.com/query?database=MyDB&retention_policy=MyRP \
   -H "Authorization: Token YourAuthToken" \
   --data-urlencode "q=SELECT used_percent FROM mem WHERE host=host1"
```

Comment:
To tie into the mapping example above, let's update this:
`FROM mem`
to this:
`FROM example-db.example-rp.example-measurement`